modules/datadog-monitor/README.md

```diff
@@ -89,29 +94,29 @@ elb-lb-httpcode-5xx-notify:
     Check LB
   escalation_message: ""
   tags: {}
+  options:
+    priority: 3
```

Comment:
```suggestion
```

Looks like `priority` is not a part of options but it's own variable

Reply:
yep! I missed that. Thanks!